beacon-chain/sync/subscriber_data_column_sidecar.go

```diff
@@ -6,27 +6,26 @@ import (
 
 	"github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed"
 	opfeed "github.com/prysmaticlabs/prysm/v5/beacon-chain/core/feed/operation"
-	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/blocks"
 	"google.golang.org/protobuf/proto"
 )
 
 func (s *Service) dataColumnSubscriber(ctx context.Context, msg proto.Message) error {
-	b, ok := msg.(*ethpb.DataColumnSidecar)
+	dc, ok := msg.(blocks.VerifiedRODataColumn)
```

Comment:
this is going to fail, our validation pipelines will pass in a `*ethpb.DataColumnSidecar`. You will need to convert the sidecar to a read only object at the end of the pipeline

Reply:
Fixed in 708be79ee3eafc204cd14d88568b7857af174261.